broker-core/src/main/java/io/zeebe/broker/exporter/stream/ExporterStreamProcessorState.java

```diff
@@ -95,7 +95,7 @@ public long getPosition(final byte[] exporterId) {
   public ExporterRecord newExporterRecord() {
     final ExporterRecord record = new ExporterRecord();
 
-    try (final RocksIterator iterator = getDb().newIterator()) {
+    try (RocksIterator iterator = getDb().newIterator()) {
```

Comment:
always fun the auto formating :sweat_smile: or do you removed the final by yourself ? :)